doc/architecture.md

```diff
@@ -58,8 +58,24 @@ The Abacus Broker supports the following flows:
       ]
     }
    ```
-   :exclamation: With the current version only one metering plan is supported. For rating and pricing default sample plans are created. The field `plan_id` is required, but not used yet.
+   :exclamation: With the current version only one metering plan is supported. For rating and pricing default sample plans are created. The field `plan_id` is required, but not used yet. In addition to the plan you could also include resource provider specific information. The resource provider section may look like this : 
+    ```json
+    {
+      "plans": [
+        {
+          "plan": {
+            ...
+          }
 
+          "resource_provider": {
+            "service_name": "ML",
+            "service_plan_name": "small"
+          }
+        }
+      ]
+    }
+   ```
+    In order to use this functionality you should implement Service plugin (API is described in the Provisioning plugin). 
```

Comment:
I think it is better to remove "Provisioning plugin" and replace it with "here" and link to the code